Stream_support/include/CGAL/IO/File_scanner_OFF.h

```diff
@@ -540,7 +540,7 @@ class CGAL_EXPORT File_scanner_OFF : public File_header_OFF {
         //split it into strings
         std::istringstream iss(col);
         //holds the rgb values
-        unsigned char rgb[3];
+        unsigned char rgb[3] {0,0,0};
```

Comment:
I prefer the following value-initialization:
```suggestion
        unsigned char rgb[3] {};
```
otherwise there might be compilers that will warn about the conversion between `int` and `unsigned char`.

The effect is the same.

Reply:
Fixed